generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -1247,10 +1247,16 @@ def create_table_scale(
                 scaled_vals=scaled_vals))
 
     def create_task(self, session_name):
+        metadata = (
+            ('ni-api-key', self._grpc_options.api_key),
+        )
         response = self._invoke(
             self._client.CreateTask,
-            grpc_types.CreateTaskRequest(session_name=session_name))
-        return response.task
+            grpc_types.CreateTaskRequest(
+                initialization_behavior=self._grpc_options.initialization_behavior,
+                session_name=session_name),
+            metadata=metadata)
+        return response.new_session_initialized, response.task
```

Comment:
The task handle is the primary result. `new_session_initialized` is secondary, so please add it to the end, not the beginning.